Xwt/Xwt/Key.cs

```diff
@@ -27,6 +27,7 @@
 
 namespace Xwt
 {
+	[Flags]
```

Comment:
So multiple `Key` can be sent as accelerator, I could have also made `Accelerator` class to take in an array of `Key`, but that is really memory inefficient compared to this.